src/client/ui/inputs/CompoundNumericInput.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Slider from "rc-slider";
+import NumericInput from "./NumericInput";
+import styles from "./CompoundNumericInput.scss";
+
+export default function CompoundNumericInput({ value, onChange, ...extras }) {
+  const { min, max } = extras;
+  const step = max && min ? (max - min) / 10 : 0.1;
```

Comment:
We can probably add step to the component definitions and pass step in through the extras property, the default step can be 0.1.